lib/common/utils.js

```diff
@@ -203,7 +203,7 @@ function cleanChildrenRoutes(routes, isChild = false) {
       if (route.children.find((child) => child.path === '')) {
         delete route.name
       }
-      route.children = cleanChildrenRoutes(route.children, true)
+      route.children = cleanChildrenRoutes(route.children, /^\/:/.test(route.path), true)
```

Comment:
I suggest adding a comment referring to the real issue number or a description about it. Comments are free but really useful when reviewing code :)

Reply:
sure.